conf/inter_athena.conf

```diff
@@ -97,6 +97,7 @@ global_acc_reg_num_table: global_acc_reg_num
 global_acc_reg_str_table: global_acc_reg_str
 
 // Char Database Tables
+achievement_table: achievement
```

Comment:
I'd prefer if you added this add the end so it matches the order in the source code.

Also we usually tend to look for recent additions at the bottom.

Reply:
Actually we should change the order to alphabetical in a future PR for easier look up